test/benzene/valgrind-integral/test

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+
+import os
+import sys
+sys.path.append(os.path.join(os.path.dirname(__file__), '..', '..'))
+
+from runtest import version_info, get_filter, cli, run
+from runtest_config import configure
+
+assert version_info.major == 2
+
+os.environ["GIMIC_LAUNCHER"] = "valgrind --tool=memcheck --log-file=valgrid-summary.dat --leak-check=full --track-origins=yes --show-reachable=yes -v"
+
+options = cli()
+
+# For valgrind with options specified
+f = [
+     get_filter(from_string='in use at exit',
+                num_lines=1,
+                rel_tolerance=0),
+     get_filter(from_string='ERROR SUMMARY',
+               to_string='errors from',
+               rel_tolerance=0),
+     ]
+#    get_filter(from_string=' *** Integrating current',
+#               num_lines=10,
+#               rel_tolerance=1.0e-8),
+#    ]
+
+# # For valgrind without options printed in the log file 
+# f = [
+#     get_filter(from_string='definitely lost',
+#                num_lines=1,
+#                rel_tolerance=0),
+#     get_filter(from_string='indirectly lost',
+#                num_lines=1,
+#                rel_tolerance=0),
+#     get_filter(from_string='possibly lost',
+#                num_lines=1,
+#                rel_tolerance=0)
+#     ]
```

Comment:
Please remove the out-commented leftovers.

Reply:
Do we want to also check the value of the integrated current in the debug tests or just the memory leak report?

Reply:
Also the correctness.